Cargo.toml

```diff
@@ -6,15 +6,16 @@ license = "Apache-2.0/MIT"
 name = "cargo-show"
 readme = "README.md"
 repository = "https://github.com/g-k/cargo-show"
-version = "0.5.0"
+version = "0.5.1"
 
 [[bin]]
 name = "cargo-show"
 path = "src/main.rs"
 
 [dependencies]
 docopt = "0.8.0"
-g-k-crates-io-client = "0.9.0"
+# g-k-crates-io-client = "0.9.0"
+g-k-crates-io-client = { git = "https://github.com/pravic/g-k-crates-io-client", branch = "update" }
```

Comment:
merged your PR and published 0.16.0: https://crates.io/crates/g-k-crates-io-client/0.16.0 so this can be updated to that version.